oracle/tests/test_custom_queries.py

```diff
@@ -107,3 +107,32 @@ def test_custom_queries_multiple_results(aggregator, check):
     aggregator.assert_metric(
         "oracle.test1.metric", value=2, count=1, tags=["tag_name:tag_value2", "query_tags1", "custom_tag"]
     )
+
+def test_custom_queries_metric_prefix_skip_column(aggregator, check):
```

Comment:
I see this test very 'implementation dependent'
Why not you create a check with this configuration and simply run the check with the right mocks?

Reply:
After talking offline, reformatted the tests to be independent of private method.